galaxy.yml

```diff
@@ -0,0 +1,70 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: styra
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: opa
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Tim Hinrichs <tim@styra.com>
+
+
+### OPTIONAL but strongly recommended
+# A short summary description of the collection
+description: Installer for Open Policy Agent, courtesy of Styra, Inc.
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- Apache-2.0
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+# license_file: ''
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags: ["policy", "openpolicyagent", "styra", "authorization", "permission", "declarative", "pdp", "policy_decision_point"]
```

Comment:
For Certification, you'll also need to add one of the following tags to this list:
`application, cloud, database, infrastructure, linux, monitoring, networking, security, storage, tools, windows`

Reply:
Done